src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -10,6 +10,9 @@
      * Результат: 21</p>
      */
     public int getFibonacciNumber(int index) {
-        return 0;
+        int result = 0;
```

Comment:
@voloshynamaryna11 Are you using this variable?

Reply:
done!